server/config.ts

```diff
@@ -30,6 +30,7 @@ export interface ApiConfig {
 }
 
 export default {
+  applicationName: 'Manage prison visits',
```

Comment:
Moving application name to here (instead of in Nunjucks setup) so it's accessible everywhere.